src/renderer/api/terminal-api.ts

```diff
@@ -119,9 +119,9 @@ export class TerminalApi extends WebSocketApi {
      * its own rc file (or RunCommands file) such as `.bashrc` or `.zshrc`.
      *
      * This heuistic assumes that the prompt line of a terminal is a single line
-     * and ends with a whitespace character.
+     * and ends with a whitespace character or the > character (windows's CMD).
```

Comment:
The prompt character is customizable so this may not be true anyway...

Reply:
That is why I consider it a heuristic. It is very real race condition when sending a command to a newly created terminal tab. 

I don't know any other way of doing it. This is an attempt to try and make sure that the connection is both alive and that the backing process (wherever it might be) is ready to receive input.

Reply:
I see. Then I think the notification should be different